portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Key;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that deletes data from the database. */
+@WebServlet("/delete-data")
+public class DeleteServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Comment").setKeysOnly();
+    PreparedQuery keys = datastore.prepare(query);
+    datastore.delete(keys.asIterable());
```

Comment:
`DatastoreService.delete` takes a `Key`, not an `Entity`. Try something like:
```java
PreparedQuery preparedQuery = datastore.prepare(query);
    Iterator<Entity> entities = preparedQuery.asIterator();
    while (entities.hasNext()) {
      Key key = entities.next().getKey();
      datastore.delete(key);	
    }
```

Reply:
Oh ok, I will do that then.